packages/perseus-core/src/data-schema.ts

```diff
@@ -1205,6 +1205,7 @@ export type PerseusNumericInputWidgetOptions = {
     // Used by examples, maybe not used and should be removed in the future
     // see TODO in numeric-input
     answerForms?: ReadonlyArray<PerseusNumericInputAnswerForm>;
+    fullAnswerForms?: ReadonlyArray<PerseusNumericInputAnswerForm>;
```

Comment:
I'm expecting that this will change (or something will need to change). Here's what I think happened:

1. Originally NumericInput used `PerseusNumericInputWidgetOptions.answerForms`
2. Then NumericInput was updated to use `PerseusNumericInputAnswer.answerForms` and `PerseusNumericInputWidgetOptions.answerForms` was deprecated (see: "maybe not used and should be removed in the future")

Now `PerseusNumericInputWidgetOptions.answers` is being removed and I think we need _something like_ `PerseusNumericInputWidgetOptions.answerForms` again **but** `PerseusNumericInputWidgetOptions.answerForms` _might exist on old data_ so I called the new thing `fullAnswerForms` pending more research.